README.md

```diff
@@ -247,7 +247,7 @@ public final class UpdateUserService {
 
 #### Renaming and retyping dependencies
 
-It is possible to rename or retype a dependency that is `@Instantiated` or `@Forwarded` by an object higher up in the dependency tree with the `@Received` macro. Renamed or retyped dependencies are able to be received with their new name and type by objects instantiated further down the dependency tree.
+It is possible to rename or retype a dependency that is `@Instantiated` or `@Forwarded` by an object higher up in the dependency tree with the `@Received(fulfilledByDependencyNamed:ofType:)` macro. Renamed or retyped dependencies are able to be received with their new name and type by objects instantiated further down the dependency tree.
```

Comment:
not directly related to the thrust of this PR, but I figured referring to the renaming macro consistently in the README was a good idea